web/src/stream_popover.js

```diff
@@ -343,13 +344,22 @@ export function build_move_topic_to_stream_popover(current_stream_id, topic_name
         });
     }
 
+    function focus_on_move_modal_render() {
+        if (!disable_stream_input && args.disable_topic_input) {
+            $("#select_stream_widget .button").trigger("focus");
+        } else {
+            ui_util.place_caret_at_end($(".inline_topic_edit")[0]);
```

Comment:
We only have single element with this class, but may still be better to use a more specific selector. I am not sure.

Reply:
Yeah, let's use a more specific selector ... or actually better might be to rename `inline_topic_edit` within this modal -- that name is a legacy detail of when it was inline in the message, but this input is in a modal now.

Or something? We use the same class name for the pencil icon in message recipient bars, so I'm not even convinced this is unique.

So maybe a prep commit to rename the use inside this form to `move_messages_edit_topic`, being careful to avoid affecting the actual inline `inline_topic_edit` (which seems to be referenced mainly as `#inline_topic_edit`).